starters/tailwind/package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "lucide-react": "^0.292.0",
     "postcss": "^8.4.31",
-    "react-aria-components": "^1.0.0-rc.0",
+    "react-aria-components": "latest",
```

Comment:
SInce this app doesn't have a lockfile, think it should already match the latest 1.* versions due to the ^ right?

Reply:
correct, though i like being more explicit about it, I always forget that this is how semver handles pre-releases
https://semver.npmjs.com/
<img width="836" alt="Screenshot 2024-05-02 at 12 28 25 PM" src="https://github.com/adobe/react-spectrum/assets/698229/0101b927-94ac-4f24-95b4-99e8f112d15f">